NitroxClient/GameLogic/Spawning/Metadata/PrecursorKeyTerminalMetadataProcessor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+using NitroxModel.Logger;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata
+{
+    public class PrecursorKeyTerminalMetadataProcessor : GenericEntityMetadataProcessor<PrecursorKeyTerminalMetadata>
+    {
+        public override void ProcessMetadata(GameObject gameObject, PrecursorKeyTerminalMetadata metadata)
+        {
+            Log.Debug($"Received precursor key terminal metadata change for {gameObject.name} with data of {metadata}");
+
+            PrecursorKeyTerminal precursorKeyTerminal = gameObject.GetComponent<PrecursorKeyTerminal>();
+            if (precursorKeyTerminal)
+            {
+                Utils.PlayFMODAsset(precursorKeyTerminal.useSound, precursorKeyTerminal.transform, 20f);
```

Comment:
As far as I remember this is also called when we players are joining. So this sound can falsely be played on connection (and if it's 2D not 3D everywhere in the world), am I right?

Reply:
Yes I beleive you are right on that one, it was something that crossed my mind anyway. Will remove till audio issues are fixed.

Reply:
Ok. I am planning to work on a global audio syncing system once I get back home from vacation.